<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21_view</class>
 <widget class="QMainWindow" name="s21_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>s21_view</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 30px;
	color: rgb(238, 116, 255);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>460</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_0{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_0:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_1{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_1:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_2{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_2:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_3{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_3:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_4{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_4:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_5{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_5:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_6{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_6:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_7{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_7:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_8{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_8:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_9{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_9:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_sin{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_sin:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>sin(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_cos{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_cos:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>cos(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_tan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_tan{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_tan:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>tan(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_asin">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_asin{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_asin:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>asin(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_acos">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_acos{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_acos:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>acos(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_atan">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>480</y>
      <width>100</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_atan{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_atan:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>atan(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_ln">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_ln{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_ln:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>ln(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_log">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_log{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_log:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>ln(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mod">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_mod{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_mod:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_sqrt">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_sqrt{
	font-size: 30px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_sqrt:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>sqrt(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_open">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_open{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_open:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_close">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_close{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_close:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_point">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>460</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_point{
	align: center;
	font-size: 70px;
	color: rgb(255, 73, 85);
	background-color: rgb(112, 84, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 20px;
}
QPushButton#button_point:pressed {
	background-color: rgb(50, 255, 245);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_plus{
	font-size: 30px;
	color: rgb(148, 17, 0);
	background-color: rgb(255, 251, 0);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_plus:pressed {
	background-color: rgb(115, 253, 255);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_minus">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>310</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_minus{
	font-size: 30px;
	color: rgb(148, 17, 0);
	background-color: rgb(255, 251, 0);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_minus:pressed {
	background-color: rgb(115, 253, 255);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mult">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_mult{
	font-size: 30px;
	color: rgb(148, 17, 0);
	background-color: rgb(255, 251, 0);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_mult:pressed {
	background-color: rgb(115, 253, 255);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_div">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>420</y>
      <width>51</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_div{
	font-size: 30px;
	color: rgb(148, 17, 0);
	background-color: rgb(255, 251, 0);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_div:pressed {
	background-color: rgb(115, 253, 255);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_pow">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>410</y>
      <width>51</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_pow{
	font-size: 30px;
	color: rgb(148, 17, 0);
	background-color: rgb(255, 251, 0);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_pow:pressed {
	background-color: rgb(115, 253, 255);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equal">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>280</y>
      <width>100</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_equal{
	font-size: 30px;
	color: rgb(1, 25, 147);
	background-color: rgb(255, 255, 255);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_equal:pressed {
	background-color: rgb(255, 126, 121);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>151</y>
      <width>100</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#button_clear{
	font-size: 30px;
	color: rgb(238, 116, 255);
	background-color: rgb(142, 255, 149);
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
QPushButton#button_clear:pressed {
	background-color: rgb(255, 77, 93);
	border-style: inset;
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>547</x>
      <y>19</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font-size: 30px;
	color: rgb(238, 116, 255);
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border: 1px solid gray;
	background-color : white;
	border-style: outset;
	border-width: 6px;
	border-color: beige;
	border-radius: 30px;
}
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_credit">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>380</y>
      <width>100</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>CREDIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
